### PR TITLE
added bad-value-for-attribute-rel-on-element-a-the-string-is-not-a-re…

### PR DESCRIPTION
**Pull request by rujaun**

### CODE REVIEW

File: compatibility/html/bad-value-for-attribute-rel-on-element-a-the-string-is-not-a-registered-keyword.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Invalid link type was found for attribute `rel` in `a` tag. HTML5 only allows you to use the standard link types without registering a new type.
+
+```html
+<a href="sample.html" rel="non-existing-linktype">Sample Link</a>
+```
+
+# How do I fix this ?
+
+Use a standard link type:
+* alternate
```
Comment (Johanndutoit) [inline compatibility/html/bad-value-for-attribute-rel-on-element-a-the-string-is-not-a-registered-keyword.md]: Any way to give a real quick description on what each of these do ? IE when should I choose what.

Comment (rujaun) [inline compatibility/html/bad-value-for-attribute-rel-on-element-a-the-string-is-not-a-registered-keyword.md]: In my own words no - but the w3 has short descriptions for each. would they need to be my own words?

Comment (Johanndutoit) [inline compatibility/html/bad-value-for-attribute-rel-on-element-a-the-string-is-not-a-registered-keyword.md]: Yes would have to your own words. We can help too so just let us know, then I'll merge and we'll collab a bit.

Comment (rujaun) [inline compatibility/html/bad-value-for-attribute-rel-on-element-a-the-string-is-not-a-registered-keyword.md]: Cool let me see what I can do

**Comment by Johanndutoit** [comment]
Merging for now, we can update with the descriptions later

**Pull request merged**